src/Raven.Server/Documents/Sharding/Subscriptions/SubscriptionConnectionsStateOrchestrator.cs

```diff
@@ -142,15 +143,17 @@ public override async Task HandleConnectionExceptionAsync(OrchestratedSubscripti
         {
             DropSubscription(new SubscriptionClosedException("Got unexpected exception, dropping workers connections.", canReconnect: true, e));
         }
+
+        await t;
```

Comment:
what happened if an exception is thrown here?

Reply:
I dont see what can throw in `DisposeWorkers()` but I can add try-catch as precaution